igor-web/igor-web.gradle

```diff
@@ -63,13 +63,3 @@ dependencies {
     testCompile 'com.github.tomakehurst:wiremock-jre8:2.22.0'
     testCompile 'org.mockito:mockito-core:2.6.8'
 }
-
-configurations.all {
-    resolutionStrategy {
-        force 'org.apache.log4j:log4j:1.2.17'
-        force 'commons-codec:commons-codec:1.7'
-    }
-    exclude group: 'javax.servlet', module: 'servlet-api'
```

Comment:
There is no `servlet-api` dependency in the dependency tree to exclude (of either group), nor an `slf4j-log4j12` dependency.